package.json

```diff
@@ -36,7 +32,14 @@
     "url": "https://github.com/babel/babel-eslint/issues"
   },
   "homepage": "https://github.com/babel/babel-eslint",
+  "peerDependencies": {
```

Comment:
This is a lot of peer deps; eslint and babel-core makes sense, but aren’t the other two addressed by babel-core?

Reply:
Yeah code-frame and babylon are dependencies of babel/core that's true. So if you are on npm 3 it should be fine?

Reply:
Since we rely on those packages directly and they could potentially not be required by @babel/core I thought it would be safer this way. Can definitely remove them if we think that's being overprotective!

Reply:
The goal is to set up the deps such that when the user runs `npm install && npm ls`, everything exits zero.

Since we rely on them directly, I'd rather them be deps. *However* we could make them *both* deps and peerDeps, and then anyone on npm3+ should get them autoinstalled and satisfy the peer dependency, and any conflicts due to changes in babel-core would cause `npm ls` to fail.